content/en/opentelemetry/otlp_endpoint.md

```diff
@@ -0,0 +1,263 @@
+
+```shell
+export OTEL_EXPORTER_OTLP_TRACES_PROTOCOL="http/protobuf"
+export OTEL_EXPORTER_OTLP_TRACES_ENDPOINT="{{< region-param key="otlp_trace_endpoint" >}}"
+export OTEL_EXPORTER_OTLP_TRACES_HEADERS="dd-protocol=otlp,dd-api-key=${DD_API_KEY},dd-otlp-source=${YOUR_ORG}"
+```
+
+#### Manual instrumentation
+
+If you are using manual instrumentation with OpenTelemetry SDKs, configure the OTLP HTTP Protobuf exporter programmatically.
+
+<div class="alert alert-info">Replace <code>${YOUR_ENDPOINT}</code> with {{< region-param key="otlp_trace_endpoint" code="true" >}}. This is based on your <a href="/getting_started/site/">Datadog site</a>, which is {{< region-param key=dd_datacenter code="true" >}}.</div>
+
+{{< tabs >}}
+{{% tab "JavaScript" %}}
+
+The JavaScript exporter is [`exporter-trace-otlp-proto`][100]. To configure the exporter, use the following code snippet:
+
+```javascript
+const { OTLPTraceExporter } = require('@opentelemetry/exporter-trace-otlp-proto');  // OTLP http/protobuf exporter
+
+const exporter = new OTLPTraceExporter({
+  url: '${YOUR_ENDPOINT}', // Replace this with the correct endpoint
+  headers: {
+    Cdd-protocol': 'otlp', 
+    'dd-api-key': process.env.DD_API_KEY,
+    'dd-otel-span-mapping': '{span_name_as_resource_name: false}',
+    'dd-otlp-source': '${YOUR_ORG}',
+  },
+});
+```
+[100]: https://www.npmjs.com/package/@opentelemetry/exporter-trace-otlp-proto
+
+{{% /tab %}}
+
+{{% tab "Java" %}}
+
+The Java exporter is [OtlpHttpSpanExporter][200]. To configure the exporter, use the following code snippet:
+
+```java
+import io.opentelemetry.exporter.otlp.http.trace.OtlpHttpSpanExporter;
+
+OtlpHttpSpanExporter exporter = OtlpHttpSpanExporter.builder()
+    .setEndpoint("${YOUR_ENDPOINT}") // Replace this with the correct endpoint
+    .addHeader("dd-protocol", "otlp")
+    .addHeader("dd-api-key", System.getenv("DD_API_KEY"))
+    .addHeader("dd-otlp-source", "${YOUR_ORG}")
+    .build();
+```
+
+[200]: https://javadoc.io/doc/io.opentelemetry/opentelemetry-exporter-otlp-http-trace/
+
+{{% /tab %}}
+{{% tab "Go" %}}
+
+The Go exporter is [otlptracehttp][300]. To configure the exporter, use the following code snippet:
+
+```go
+import "go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp"
+
+traceExporter, err := otlptracehttp.New(
+	ctx,
+	otlptracehttp.WithEndpoint("${YOUR_ENDPOINT}"), // Replace this with the correct endpoint
+	otlptracehttp.WithURLPath("/api/v0.2/traces"),
+	otlptracehttp.WithHeaders(
+		map[string]string{
+			"dd-protocol": "otlp", 
+			"dd-api-key": os.Getenv("DD_API_KEY"),
+			"dd-otel-span-mapping": "{span_name_as_resource_name: false}",
+                  "dd-otlp-source": "${YOUR_ORG}",
+		}),
+)
+```
+
+[300]: http://go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp
+
+{{% /tab %}}
+{{% tab "Python" %}}
+
+The Python exporter is [OTLPSpanExporter][400]. To configure the exporter, use the following code snippet:
```

Comment:
```suggestion
The Java exporter is [`OtlpHttpSpanExporter`][200]. To configure the exporter, use the following code snippet:

```java
import io.opentelemetry.exporter.otlp.http.trace.OtlpHttpSpanExporter;

OtlpHttpSpanExporter exporter = OtlpHttpSpanExporter.builder()
    .setEndpoint("${YOUR_ENDPOINT}") // Replace this with the correct endpoint
    .addHeader("dd-protocol", "otlp")
    .addHeader("dd-api-key", System.getenv("DD_API_KEY"))
    .addHeader("dd-otlp-source", "${YOUR_ORG}")
    .build();
```

[200]: https://javadoc.io/doc/io.opentelemetry/opentelemetry-exporter-otlp-http-trace/

{{% /tab %}}
{{% tab "Go" %}}

The Go exporter is [`otlptracehttp`][300]. To configure the exporter, use the following code snippet:

```go
import "go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp"

traceExporter, err := otlptracehttp.New(
	ctx,
	otlptracehttp.WithEndpoint("${YOUR_ENDPOINT}"), // Replace this with the correct endpoint
	otlptracehttp.WithURLPath("/api/v0.2/traces"),
	otlptracehttp.WithHeaders(
		map[string]string{
			"dd-protocol": "otlp", 
			"dd-api-key": os.Getenv("DD_API_KEY"),
			"dd-otel-span-mapping": "{span_name_as_resource_name: false}",
                  "dd-otlp-source": "${YOUR_ORG}",
		}),
)
```

[300]: http://go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp

{{% /tab %}}
{{% tab "Python" %}}

The Python exporter is [`OTLPSpanExporter`][400]. To configure the exporter, use the following code snippet:
```

Reply:
Apparently Github can't handle this suggestion. But I'm just trying to put all the package/module names in code font